provisions/roles/jenkins/slave/tasks/setup_worker.yml

```diff
@@ -19,48 +19,49 @@
   file: dest=/opt/cccp-service/beanstalk_worker state=directory
 
 - name: Build container-pipeline image
-  shell: docker build -t container-pipeline -f ./Dockerfile .
-  args:
-      chdir: /opt/cccp-service
+  sudo: yes
```

Comment:
this wont have effect if user is `root`, but it enables our service to be deployed on environments where sudo access is not given to devs.